src/components/Karma/CircleGradient.tsx

```diff
@@ -11,7 +11,7 @@ export const CircleGradient = (props: {
   children: React.ReactNode;
   testId?: string;
 }) => {
-  const r = props.size / 2 - props.width;
+  const r = props.size / 2 - props.width / 1.9;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the calculation of `r` might affect the visual appearance of the component. Ensure that this change is intentional and tested across different screen sizes.
```diff
-  const r = props.size / 2 - props.width;
+  const r = props.size / 2 - props.width / 1.9;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->